src/code/z_bgcheck.c

```diff
@@ -24,6 +24,14 @@
 #define COLPOLY_IGNORE_ENTITY (1 << 1)
 #define COLPOLY_IGNORE_PROJECTILES (1 << 2)
 
+// floor raycast flags (frcFlags)
+#define BGCHECK_FRC_0 0
```

Comment:
We usually don't have a flag define for 0, and this one isn't even used so I think we should just omit it
```suggestion
```